### DISCUSSION TITLE
Openupgrade with doodba

### DISCUSSION THREAD

**Discussion by carlosecv** [Tecnativa/doodba#410] [Q&A] [answered]
¿Do someone has done OpenUpgrade with doodba?
¿Any link with instructables?

Thanks

**Comment by pedrobaeza** [comment, marked as answer]
Just put such remote in ./odoo section. Please use questions for this, not issues.

**Reply by carlosecv** [reply]
Thank you

**Comment by carlosecv** [comment]
Im trying to do my first migration from odoo v9 to odoo v10, so i have custom module on version 9 has custom gpc_country_id field that is related to country table on sale_order so how do i tell my module that country table has being changed to other table on ver 10?

This gives me the try of upgrade from version 9 to 10

IntegrityError: update or delete on table "res_country" violates foreign key constraint "sale_order_gpc_country_id_fkey" on table "sale_order"
DETAIL:  Key (id)=(265) is still referenced from table "sale_order".

I apologize if this is not the correct project to ask this but on OpenUpgrade there is no Discussion forum where to do this kind of basic questions.

**Reply by joao-p-marques** [reply]
Hi!

Indeed, this is not the place to ask this sort of questions, so I am afraid we can't help you here...

Perhaps you can check the official OCA Discord / IRC / Matrix rooms? 
You can find more information here:
https://matrix.to/#/+oca:matrix.org 
https://odoo-community.org/page/Contribute (for the Discord Server and IRC rooms)
